adbacli.py

```diff
@@ -112,8 +112,8 @@
     try:
         connection.auth(args.username, args.password)
     except Exception as e:
-        print('Exception: %s', e)
-        logging.debug('Exception Occurred: ' + str(e))
+        print('Exception: %s' % e)
```

Comment:
This same error is in most of the exceptions.

Reply:
Aah yes i see. Ill also change those.